tests/regression/results.csv

```diff
@@ -11,10 +11,10 @@ silesia.tar,                        level 6,                            compress
 silesia.tar,                        level 7,                            compress simple,                    4613541
 silesia.tar,                        level 9,                            compress simple,                    4555426
 silesia.tar,                        level 13,                           compress simple,                    4491764
-silesia.tar,                        level 16,                           compress simple,                    4356834
-silesia.tar,                        level 19,                           compress simple,                    4264392
```

Comment:
Investigating why this regression happens (and compress simple and compress cctx is the only ones that have these large regressions).

Reply:
`uncompressed literals optimal` is also regressing. I suspect that you maybe are writing beyond the end of your workspace?

Reply:
It seems to just have been that I never ran `make clean` in the regression test when I still was on the revision with the correctness bug in the insertion sort.